packages/host/app/services/realm-info-service.ts

```diff
@@ -99,12 +100,11 @@ export default class RealmInfoService extends Service {
       if (this.cachedRealmInfos.has(realmURLString)) {
         return this.cachedRealmInfos.get(realmURLString)!;
       } else {
-        let realmInfoResponse = await this.loaderService.loader.fetch(
-          `${realmURLString}_info`,
-          { headers: { Accept: SupportedMimeType.RealmInfo } },
-        );
+        let realmInfoResponse = await this.fetch(`${realmURLString}_info`, {
+          headers: { Accept: SupportedMimeType.RealmInfo },
+        });
 
-        let realmInfo = (await realmInfoResponse.json())?.data
+        let realmInfo = (await realmInfoResponse.clone().json())?.data
```

Comment:
I’m curious, why do we clone this?

Reply:
Avoiding the `json body is already read` error. I'll add a comment.

Reply:
Added [1e1da51](https://github.com/cardstack/boxel/pull/1259/commits/1e1da5101b8bf2bc14d3f4d4f311443551b22329)